controllers/search_controller.go

```diff
@@ -62,6 +65,12 @@ func (r *SearchReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		}
 		return reconcile.Result{}, err
 	}
+
+	// Do not reconcile objects if this instance of search is labeled "paused"
+	if IsPaused(instance.GetAnnotations()) {
+		log.Info("Search reconciliation is paused. Nothing more to do.")
```

Comment:
- Avoid the word "Search" in case we ever need to rename this.
- Should we add a hint to help resume the operator?
```suggestion
		log.Info("Reconciliation is paused because the annotation 'operator-pause: true' was found.")
```

Reply:
Done.